static/javascript/launch.js

```diff
@@ -497,12 +516,14 @@ function loadSchemaMetadata(schemaName, schemaUrl, cirInstrumentId) {
               })()}</div>`;
             })
             .join("");
-        updateSDSDropdown();
+        if (!window.location.href.includes("test-launcher.eq.gcp.onsdigital.uk")) {
```

Comment:
Not sure we can just hardcode the test launcher url here - this probably needs to be configurable, as there may be other envs that could be impacted. For example, author raised an issue not being able to use launcher in their Preprod environment, as they don't have SDS running.

Reply:
Yeah I agree I thought there could be a better solution but as this was a dev tool I thought this would be good enough for now and could be changed later if we needed to add more envs to this list